seed/views/v3/organizations.py

```diff
@@ -850,6 +854,15 @@ def get_raw_report_data(self, organization_id, access_level_instance, cycles, x_
             .order_by("id")
         )
 
+        if filter_group_id:
+            filter_group = FilterGroup.objects.get(pk=filter_group_id)
+            if filter_group.query_dict:
+                qd = QueryDict(mutable=True)
+                qd.update(filter_group.query_dict)
+                columns = Column.retrieve_all(org_id=organization_id, inventory_type="property", only_used=False, include_related=False)
+                filters, _annotations, _order_by = build_view_filters_and_sorts(qd, columns, "property")
+                all_property_views = all_property_views.filter(filters)
+
```

Comment:
Does this handle the filter group's label filtering too? I think you have to do something like in[ this function ](https://github.com/SEED-platform/seed/blob/Fix-element-statistics/seed/models/data_views.py#L31), I I bet you can get it a little cleaner

Reply:
It sure doesn't.  Thanks for pointing that out.